tests/Doctrine/Tests/Common/Cache/MongoDBCacheTest.php

```diff
@@ -6,20 +6,34 @@
 use Doctrine\Common\Cache\MongoDBCache;
 use MongoClient;
 use MongoCollection;
+use MongoConnectionException;
 
 /**
  * @requires extension mongodb
  */
 class MongoDBCacheTest extends CacheTest
 {
+    private static $skip;
+
     /**
      * @var MongoCollection
      */
     private $collection;
 
     protected function setUp()
     {
-        $mongo = new MongoClient();
+        if (static::$skip === true) {
```

Comment:
Also: speed is not a problem on this project: if you want to work on that, we have some tests using `sleep()` 😅